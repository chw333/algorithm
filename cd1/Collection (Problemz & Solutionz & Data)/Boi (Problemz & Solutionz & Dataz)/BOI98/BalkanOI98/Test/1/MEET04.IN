176 77 161
9 2 8
10 6 9
15 14 2
21 1 7
24 19 9
32 13 2
33 11 6
35 21 9
36 30 4
39 26 9
43 20 7
44 34 4
47 3 3
49 33 3
51 14 7
52 3 8
57 7 9
58 5 2
58 8 6
60 25 2
61 30 3
61 38 7
63 15 8
66 11 5
67 55 8
68 3 9
70 62 2
70 69 4
72 31 8
73 2 5
74 49 5
75 12 9
75 64 7
76 41 8
77 52 2
79 26 2
82 40 2
82 50 5
84 42 5
84 46 5
85 22 1
86 23 5
86 42 9
86 45 9
86 56 2
87 4 8
87 62 2
89 76 9
90 59 1
90 80 9
90 89 8
91 34 8
92 35 5
93 71 5
94 18 9
94 29 2
95 28 6
95 77 7
97 14 3
98 7 2
98 74 8
99 8 9
100 54 9
100 65 6
101 24 8
101 27 2
101 81 5
103 81 4
104 17 6
105 52 10
105 61 8
105 71 5
105 100 9
106 84 6
107 78 8
108 75 5
109 82 5
110 90 6
110 99 7
111 75 8
112 11 4
112 97 6
114 55 8
114 77 8
115 5 7
115 55 4
116 13 7
117 1 7
117 27 7
117 32 10
117 38 7
118 59 5
118 96 2
118 117 7
119 16 8
119 83 2
121 87 5
121 102 9
122 100 2
123 16 6
125 32 8
125 77 5
126 112 3
127 87 4
127 108 2
128 71 4
129 5 5
129 63 6
130 119 5
131 20 9
131 86 3
132 37 6
132 49 10
134 91 6
135 110 9
137 72 1
137 118 7
138 123 9
139 6 9
140 78 4
140 94 4
141 35 5
141 64 4
141 83 3
141 107 2
141 133 4
142 10 3
142 25 5
143 80 3
144 111 1
145 51 9
145 95 2
146 115 4
147 39 8
147 48 4
147 104 5
148 19 2
149 48 6
149 53 10
150 61 4
151 26 7
151 100 9
152 36 8
152 125 3
152 139 3
153 23 7
153 34 7
153 88 3
154 40 6
155 109 7
155 144 8
156 95 4
157 101 6
158 78 3
158 124 3
159 7 7
161 13 2
161 52 4
161 85 9
161 114 5
161 125 7
161 160 3
162 114 2
162 120 2
163 155 3
164 136 10
165 2 2
165 113 7
165 148 2
166 13 7
167 88 10
167 111 7
168 73 10
169 73 9
169 164 2
170 74 5
