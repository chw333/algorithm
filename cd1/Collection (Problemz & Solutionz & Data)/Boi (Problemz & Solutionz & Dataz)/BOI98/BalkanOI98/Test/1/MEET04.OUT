77 95 145 51 14 15 63
161 114 55 115 5 129 63
